Terminal.Gui/Views/TextField.cs

```diff
@@ -854,6 +901,51 @@ int WordBackward (int p)
 			return -1;
 		}
 
+		void ShowContextMenu ()
+		{
+			ContextMenu = new ContextMenu (this,
+				new MenuBarItem (new MenuItem [] {
+					new MenuItem ("_Select All", "", () => SelectAll (), null, null, GetKeyFromCommand (Command.SelectAll)),
```

Comment:
By adding `ContextMenu` support to `TextField` this way you've raised an interesting question:

- Should built-in strings within the `Terminal.Gui` library be in English or stored in a way that is easily localizable?

Where else within the library do we have English text like this?

Reply:
Well that's will be a challenge.

Reply:
resx files would normally be how you set the library up for translations.  It uses the running systems culture to pick which resx file resource to use for the given language.  But you would need the translations.

https://docs.microsoft.com/en-us/dotnet/core/extensions/localization#resource-files

The only place I can think of currently where we use hard coded English is `OpenDialog`

Reply:
Done! https://github.com/migueldeicaza/gui.cs/pull/1615/commits/b867ab62fe080cb4f53f002a40772183369f022a